packages/driver/src/cy/commands/sessions/manager.ts

```diff
@@ -283,41 +294,34 @@ export default class SessionsManager {
       return results
     },
 
-    clearStorage: async () => {
+    clearStorage: async (type?: InternalCypress.StorageType) => {
       const origins = await this.getAllHtmlOrigins()
```

Comment:
We can expand this to accept `currentOrigin` and use `mapOrigins()` instead of `await this.getAllHtmlOrigins()` to allows specific origins to be specified as an option to cy.clearLocalStorage()

Reply:
I think that's out of scope for adding these commands.